src/server/router/peaRouter/peaRouter.ts

```diff
@@ -162,15 +161,15 @@ peaRouter.get('/:peaId/getServerSettings', (req: Request, res: Response) => {
 });
 
 /**
- * @api {post} /updateSettings
- * @apiName PostSettings
+ * @api {post} /updateServerSettings
+ * @apiName PostServerSettings
  * @apiGroup PEAController
- * @apiParam {ServerSettingsOptions}  [options]
+ * @apiParam {ServerSettingsOptions} options
  */
-peaRouter.post('/updateSettings', asyncHandler(async (req: Request, res: Response) => {
+peaRouter.post('/updateServerSettings', asyncHandler(async (req: Request, res: Response) => {
 	const manager: ModularPlantManager = req.app.get('manager');
 	try{
-		manager.updateServerSettings(req.body.options);
+		manager.updateServerSettings(req.body);
 		res.status(200).send('"'+'Success!'+'"');
```

Comment:
What exactly is the success --> please be a bit more precise on the context of the message, to give a hint what was successful

There should be a hint on the error source, may followed by the message

Reply:
you're absolute right. i was just putting it as dummy everywhere.